charts/dave/charts/eai/values-openshift.yaml

```
@@ -0,0 +1,12 @@
+# Additional values for dave-eai for openShift environments.
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)